### ISSUE TITLE
Something is wrong with rvn

### ISSUE THREAD

**Issue by davodavodavo3** [bzminer/bzminer#170]
Hi friends, seems in v13.0.1 something changed that affected kawpow algo.

flypool used

When mining rvn i got 20 accepted shares and then 20 rejected. Aftre restarting few times rejected shares are more than accepted 

Mining Rvn on v12.2.0 work more than correct without a bunch of rejected shares

**Comment by Canefis** [comment]
Yes, confirmed, all releases after that kawpow algo f@cked up completely....
